pootle/apps/pootle_statistics/models.py

```diff
@@ -232,10 +231,10 @@ def get_submission_info(self):
                                                           check_name),
                     'checks_url': reverse('pootle-checks-descriptions'),
                 })
-
-        if (self.suggestion and
-            self.type in (SubmissionTypes.SUGG_ACCEPT,
-                          SubmissionTypes.SUGG_REJECT)):
+        is_suggestion = (
+            self.suggestion
+            and (self.type == SubmissionTypes.SUGG_ACCEPT))
```

Comment:
in this case - i prefer a little extra bracket as i feet it makes the condition clearer